index.spec.js

```diff
@@ -0,0 +1,72 @@
+const request = require('supertest');
+const server = require('./server');
+const db= require('./data/dbConfig');
+
+
+describe('The APIs Endpoints', function() {
+
+    describe('GET / games route', () => {
+
+        it('should return a status code of 200', async () => {
+            let response = await request(server).get('/games');
+            expect(response.status).toBe(200);
+        });
+
+        it('should always return an array type even if an empty array', async () => {
+            const response = await request(server).get('/games');
+            expect(Array.isArray(response.body)).toBeTruthy();
+        });
+
+        it('should always not return an string data ', async () => {
+            const response = await request(server).get('/games');
+            expect(response.type).not.toMatch(/html/i);
+        });
+        
+        it('should return JSON date', async () => {
+            let response = await request(server).get('/games');
```

Comment:
Instead of making the same request to the database multiple times, you should use a `beforeAll` to make this section of tests more performant! For example:
```javascript
describe('GET / games route', () => {
  let response;
  beforeAll(async () => {
    response = await request(server).get('/games');
  });

  it('should return a status code of 200', () => {
    expect(response.status).toBe(200);
  });

  it('should return an array', () => {
    expect(Array.isArray(response.body)).toBeTruthy();
  });

  // ...
```